doc/cc65.sgml

```diff
@@ -726,6 +728,26 @@ This cc65 version has some extensions to the ISO C standard.
         will give the high byte of any unsigned value.
         <p>
 
+<item>  There is pseudo variable named <tt/__A__/ that indicates accumulator register. 
+        It can be used to highlight the returned variable in non-void
+        function where result is hidden in the <tt/asm/ section.
+        For example, in the function below
+
+        <tscreen><verb>
+        BYTE get_key_via_atarixl_romcall_device_k(void)      \
+            asm("LDA $E425");   \
+            asm("PHA");         \
+            asm("LDA $E424");   \
+            asm("PHA");         \
+            asm("RTS");         \
+            return __A__;
+        </verb></tscreen>
+
+        <tt/return __A__;/ will indicate that result is stored in the accumulator
+        and suppress warning that function has no return statement. Instead, you can
+        also use <tt/#pragma warn (return-type, off) ... pragma warn (return-type, on)/
+        <p>
+
```

Comment:
Thanks for the effort to introduce this section. However, the content seems to specific to me. Please remove it from this pull request. I see two options:

1. Just don't care about documenting `__A__` as this lack of documentation isn't really related to the topic at hand.
2. Simply adjust the section above to not only talk about `__AX__` and `__EAX__` but rather about `__A__`, `__AX__` and `__EAX__`. There's no need to extend the example.